modules/internal/ChapelDomain.chpl

```diff
@@ -1198,13 +1198,13 @@ module ChapelDomain {
 
     /* Return true if this is a stridable domain */
     // deprecated by Vass in 1.31 to implement #17131
-    //RSDW: @deprecated("domain.stridable is deprecated; use domain.strides instead")
+    @deprecated("domain.stridable is deprecated; use domain.strides instead")
     proc stridable param where this.isRectangular() {
       return _value.strides.toStridable();
```

Comment:
same on these, can you remove the newline between the attribute and function

Now that I look closer, looks like maybe a `sed` error

Reply:
Same on these.